### ISSUE TITLE
Only consider extensions in the `mad` and `scanners` reviewer queues

### ISSUE THREAD

**Issue by diox** [mozilla/addons-server#19430]
The `mad` and `scanners` queues in reviewer tools currently consider all add-on types. Their queries would likely be twice as fast if they only had to consider extensions (because they need to look at all non-disabled/deleted add-ons - scanners can flag any version of any public/non public add-on, in theory). The drawback would be, obviously, that non extensions would no longer show up in these queues. IF that's not a problem, let's do it to improve performance of these queues.

**Comment by diox** [comment]
@wagnerand to decide whether or not we should do that.

**Comment by wagnerand** [comment]
@diox Just to double check: Do we run MAD or scanners for non-extensions? If no, then let's do this.

**Comment by diox** [comment]
We do run them, except for `MAD` we skip add-ons with no code, effectively ignoring non-extensions. 

But it's technically possible we'd have say, a theme trigger a scanners rule from the manifest contents. If you think we'll never need that, we could make them skip scanners entirely.

**Comment by wagnerand** [comment]
So let's say a scanner rule (accidentally or on purpose) matches a non-extension and prevents auto-approval. How would we find out?

**Comment by diox** [comment]
We wouldn't, unless notified by the developer or when looking for this in logs.

What I'm proposing here is clearly a quick hack. The way I see it, our options are:
- Leave non-extensions in reviewer queues (do nothing, close this issue)
- Ignore non-extensions in reviewer queues, leaving the edge case possibility for scanner rules to affect them anyway with no direct way for us to find out anymore
- Ignore non-extensions in reviewer queues, removing the possibility for scanner rules to affect them entirely

**Comment by wagnerand** [comment]
> Ignore non-extensions in reviewer queues, removing the possibility for scanner rules to affect them entirely

Let's go ahead with this option.

**Comment by ioanarusiczki** [comment]
@diox 

 I did not see non-extensions displayed in these queues.

I submitted themes, lang packs, dicts with rules which were scanned by all scanners -> as shown by the [admin](https://addons-internal.stage.mozaws.net/en-US/admin/models/scanners/scannerresult/?has_matched_rules=all). Then checked the rev tools queues -> they are not displayed here

Also I 've submitted extensions. -> which showed up in scanners and/or  mad queue.

Verified on AMO stage.
